<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AlbumsPage</class>
 <widget class="QWidget" name="AlbumsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>204</width>
    <height>151</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item row="0" column="0" colspan="4">
    <widget class="LineEdit" name="search">
     <property name="toolTip">
      <string/>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="4">
    <widget class="QListView" name="view">
     <property name="dragEnabled">
      <bool>true</bool>
     </property>
     <property name="dragDropMode">
      <enum>QAbstractItemView::NoDragDrop</enum>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::ExtendedSelection</enum>
     </property>
     <property name="movement">
      <enum>QListView::Static</enum>
     </property>
     <property name="resizeMode">
      <enum>QListView::Adjust</enum>
     </property>
     <property name="layoutMode">
      <enum>QListView::Batched</enum>
     </property>
     <property name="viewMode">
      <enum>QListView::IconMode</enum>
     </property>
     <property name="uniformItemSizes">
      <bool>true</bool>
     </property>
     <property name="batchSize">
      <number>50</number>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QToolButton" name="libraryUpdate"/>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="genreCombo">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QToolButton" name="addToPlaylist"/>
   </item>
   <item row="2" column="3">
    <widget class="QToolButton" name="replacePlaylist"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LineEdit</class>
   <extends>QLineEdit</extends>
   <header>lineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
